src/services/routeConfig.js

```diff
@@ -8,6 +8,7 @@ import TestsPage from '../pages/testsPage';
 import DisciplinePage from '../pages/disciplinePage';
 import TestInfo from '../components/TestInfo';
 import TestSolution from '../components/TestSolution';
+import ErrorPage from '../pages/errorPage/errorPage';
```

Comment:
import ErrorPage from '../components/ErrorPage'

Reply:
pages should be in pages folder

Reply:
maybe a page should be in the pages folder?